x-pack/plugins/serverless/public/plugin.tsx

```diff
@@ -61,7 +60,6 @@ export class ServerlessPlugin
     }
 
     core.chrome.setChromeStyle('project');
-    management.setIsSidebarEnabled(false);
```

Comment:
I had to remove dependency on management. Specific serverless project plugins will now set this instead